api/src/tests/unit/controllers/AssetController.test.ts

```diff
@@ -1,35 +1,48 @@
 import { NextFunction, Request, Response } from "express";
 import AssetController from "../../../controllers/AssetController";
-import { expectNonFinal, mockNext, mockRequest, mockResponse } from "../../helpers/mockExpress";
+import {
+  expectNonFinal,
+  mockNext,
+  mockRequest,
+  mockResponse,
+} from "../../helpers/mockExpress";
 import ErrorController from "../../../controllers/ErrorController";
-import { testAsset } from "../../helpers/testData";
-import { AssetAttributes } from "../../../utils/types/attributeTypes";
+import { testAsset, testPaginationAssets } from "../../helpers/testData";
 import AssetService from "../../../services/AssetService";
 import Asset from "../../../database/models/asset.model";
 import { MockedLogger, resetMockLogger } from "../../helpers/mockLogger";
 import Logger from "../../../utils/Logger";
+import { PaginationResult } from "../../../utils/Paginator";
 
 jest.mock("../../../services/AssetService.ts");
 jest.mock("../../../services/BaseService.ts");
-jest.mock("../../../utils/Logger.ts", (): MockedLogger => ({
-  info: jest.fn(),
-  error: jest.fn(),
-}));
+jest.mock(
+  "../../../utils/Logger.ts",
+  (): MockedLogger => ({
+    info: jest.fn(),
+    error: jest.fn(),
+  })
+);
 
 const logger: MockedLogger = Logger as unknown as MockedLogger;
 
-describe('getAllAssets', () => {
+describe("getAllAssets", () => {
   const assetController: AssetController = new AssetController();
   let request: Request;
   let response: Response;
   let next: NextFunction;
-  let findAllMock: jest.MockedFunction<() => Promise<AssetAttributes[]>>;
+  let findAllMock: jest.MockedFunction<
```

Comment:
W (Pedantic): `findAllPaginatedMock`, to reflect the name of the method which is being mocked.

Reply:
Commit: 02dbeb7a0fa4367798ed906384e2bba6522c1763